go.mod

```diff
@@ -12,13 +12,12 @@ require (
 	github.com/ethereum/go-ethereum v1.10.18
 	github.com/ferranbt/fastssz v0.0.0-20220103083642-bc5fefefa28b
 	github.com/goccy/go-yaml v1.9.5
-	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/jonboulle/clockwork v0.3.0
 	github.com/jsternberg/zap-logfmt v1.2.0
-	github.com/libp2p/go-libp2p v0.18.0
-	github.com/libp2p/go-libp2p-core v0.15.1
-	github.com/libp2p/go-libp2p-noise v0.4.0
+	github.com/libp2p/go-libp2p v0.20.0
+	github.com/libp2p/go-libp2p-core v0.16.1
+	github.com/libp2p/go-libp2p-noise v0.5.0
```

Comment:
replace this package with github.com/libp2p/go-libp2p/p2p/security/noise where ever used as this one is deprecated